core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/kit/KitDescriptor.java

```diff
@@ -79,6 +83,12 @@
   @OneToMany(targetEntity = KitDescriptorChangeLog.class, mappedBy = "kitDescriptor")
   private final Collection<ChangeLog> changelog = new ArrayList<>();
 
+  @ManyToMany(targetEntity = TargetedSequencing.class)
+  @JoinTable(name = "TargetedSequencing_KitDescriptor", inverseJoinColumns = {
```

Comment:
The JoinTable should only be defined on one end of the relationship (here or in TargetedSequencing). Since this is a bidirectional many-to-many, you have to choose which side is the "owner" of the relationship.

The owner keeps the `@JoinTable` definition and modifying its collection will cause relationship updates. The non-owner uses the `mappedBy` property of `@ManyToMany` and modifying its collection has no effect.

The other option is to make this a unidirectional relationship. It's useful to get TargetedSequencing options for a KitDescriptor, but is it useful to get KitDescriptors for a TargetedSequencing? If not, just remove the kitDescriptors field from TargetedSequencing.

Unidirectional relationships are simpler and in my opinion should be preferred unless bidirectional is really required